### ISSUE TITLE
Dimming not setting properly

### ISSUE THREAD

**Issue by isaacrlevin** [UselessMnemonic/OpenWiz#2]
Hey there! I assume this is more a light issue than an issue with the library, but I wanted to see if you have this experience. I can update my dimming just fine, but when it is set to 100, it is nowhere near full brightness, and anything more than that gives an error. Here is some sample code to work with.

```csharp
var command = new WizParams();
command.State = true;
command.Dimming = 150;//_options.LightSettings.Wiz.Brightness;
command.Speed = 0;
command.R = 255;
command.B = 0;
command.G = 0;

UpdateLight(command, lightMacAddress);

private WizResult UpdateLight(WizParams wizParams, string lightId)
{
    WizSocket socket = new WizSocket();
    socket.GetSocket().EnableBroadcast = true; // This will enable sending to the broadcast address
    socket.GetSocket().ReceiveTimeout = 1000; // This will prevent the demo from running indefinitely
    WizHandle handle = new WizHandle(lightId, IPAddress.Broadcast); // MAC doesn't matter here

    WizState state = new WizState
    {
        Method = WizMethod.setPilot,
        Params = wizParams
    };

    socket.SendTo(state, handle);

    WizResult pilot;
    while (true)
    {
        state = socket.ReceiveFrom(handle);
        pilot = state.Result;
        break;
    }
    return pilot;
}
```

**Comment by UselessMnemonic** [comment]
Hi @isaacrlevin ! I swear I check up regularly, so sorry for the wait. I'll try to get the fix up today.

**Comment by UselessMnemonic** [comment]
So after a bit of finessing, I realized that the hue LEDs aren't as perceivably bright as the cool/warm white LEDs.

I played with Params.Dimming (by the way, values here are in [10, 100]) and no max colored LED produces as intense a light. You could try using the white LED's in tandem, but the final color looses a lot of saturation.

**Comment by isaacrlevin** [comment]
Makes sense, except why does the app allow you to have full brightness when you change color

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by UselessMnemonic** [comment]
The controls on the app map more or less directly to the parameters for `setPilot`. I went ahead and selected full bright red on the app. This is part of the result of `getPilot`:
```json
{
  "method" : "getPilot",
  "result" : {
    "rssi" : -54,
    "state" : true,
    "sceneId" : 0,
    "r" : 255,
    "dimming" : 100
  }
}
```
I tried manually setting the same color, and I get the same result. Full brightness on the app doesn't correlate to physical brightness.

**Comment by isaacrlevin** [comment]
That is really odd. If I set dimming to 100, like below, the light is nowhere near as bright as setting red in the app

```csharp
WizSocket socket = new WizSocket();
socket.GetSocket().EnableBroadcast = true; // This will enable sending to the broadcast address
socket.GetSocket().ReceiveTimeout = 1000; // This will prevent the demo from running indefinitely
WizHandle handle = new WizHandle(lightId, IPAddress.Broadcast); // MAC doesn't matter here

WizState state = new WizState
{
    Method = WizMethod.setPilot,
    Params = new WizParams{
    	 State = true,
		 Dimming = 100,
		 Speed = 0,
		 R = 255,
         B = 0,
         G = 0
    }
};

socket.SendTo(state, handle);

WizResult pilot;
while (true)
{
    state = socket.ReceiveFrom(handle);
    pilot = state.Result;
    break;
}
```

**Comment by UselessMnemonic** [comment]
Which particular bulb are you using? I have tested only with the BR30 model, so perhaps your bulb needs some extra configuration.

Alternatively, how often do you send updates? I find that frequent updates cause the bulb to stop mid-transition.

**Comment by isaacrlevin** [comment]
I have an A19 bulb

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by UselessMnemonic** [comment]
Could you post a link to the item you bought, by chance? I see a few labeled A19, I just want to be sure I grab the same one.

**Comment by isaacrlevin** [comment]
https://www.homedepot.com/p/WiZ-60W-Equivalent-A19-Colors-and-Tunable-White-Wi-Fi-Connected-Smart-LED-Light-Bulb-IZ0126081/303142930

**Comment by isaacrlevin** [comment]
Any update here?

**Comment by UselessMnemonic** [comment]
Hi there, sorry about the wait, I've gotten busier with work and I'm having troubles securing a bulb

**Comment by isaacrlevin** [comment]
Did a bit more digging here. It seems that if you increase `Dimming` to over 100, you will get a an *Invalid params* message

![image](https://user-images.githubusercontent.com/8878502/113751323-b1742c00-96c0-11eb-8da2-0dc95000ce5b.png)


Since 100 dimming is pretty dark, I think the issue is related to not accepting more than 100. For instance, I tried doing this in Python with [pywizlight](https://github.com/sbidy/pywizlight) and had no issues setting the dimming to 255. 

**Disregard the nonsense about dimming over 255, I realized that that pywizlight also uses 0-100**

**Comment by isaacrlevin** [comment]
@UselessMnemonic now that you have a light. Can you try to repro what I am seeing here?

**Comment by djniuss** [comment]
Same problem with A19 bulb

**Comment by isaacrlevin** [comment]
Hey @UselessMnemonic coming back here to see if you have had a chance to look at this. I can try to debug, but I am not sure how you debug this issue

**Comment by Tungsten66** [comment]
Hello @UselessMnemonic checking to see if had any ideas with the dimming of the A19 bulb
